script.js

```diff
@@ -0,0 +1,178 @@
+// Assignment Code
+var generateBtn = document.querySelector("#generate");
+
+// Arrays
+const numArray = ["0", "1", "2", "3", "4", "5", "6", "7", "8", "9"];
+const specialArray = [
+  "!",
+  "£",
+  "$",
+  "%",
+  "^",
+  "&",
+  "*",
+  "(",
+  ")",
+  "_",
+  "-",
+  "+",
+  "=",
+  "?",
+  "@",
+  "#",
+  "~",
+  "|",
+  "/",
+  ".",
+  ",",
+  "<",
+  ">",
+  '"',
+  ":",
+  " ",
+  ";",
+  "`",
+  "¬",
+];
+const lowerCaseArray = [
+  "a",
+  "b",
+  "c",
+  "d",
+  "e",
+  "f",
+  "g",
+  "h",
+  "i",
+  "j",
+  "k",
+  "l",
+  "m",
+  "n",
+  "o",
+  "p",
+  "q",
+  "r",
+  "s",
+  "t",
+  "u",
+  "v",
+  "w",
+  "x",
+  "y",
+  "z",
+];
+
+// convert lower case to uppercase array
+const upperCase = lowerCaseArray.join("").toUpperCase();
+const upperCaseArray = upperCase.split("");
+
+// get criteria
+// Function - prompt user to select password characteristics
+const getCriteria = function () {
+  // created a variable x which will hold the users numerical input
+  const x = prompt("Choose Password length");
```

Comment:
As mentioned in the feedback, I probably would of mentioned here about the character limits, just to give the user a heads up so they don't have to hit the alert to find out this information.